lib/msal-node/src/index.ts

```diff
@@ -36,6 +36,8 @@ export { UsernamePasswordClient } from "./client/UsernamePasswordClient.js";
 
 export {
     Configuration,
+    ManagedIdentityConfiguration,
+    ManagedIdentityIdParams,
```

Comment:
This is a bugfix needed for the sample to run. These should have already been exported here.

Reply:
Does this mean we need to release another MSAL preview @Robbie-Microsoft ?

Reply:
No. Resolving per our offline discussion.